translate/translate.js

```diff
@@ -107,6 +107,40 @@ function translateText (input, target) {
 }
 // [END translate_translate_text]
 
+// [START translate_text_with_model]
+function translateTextWithModel (input, target, model) {
+  if (!Array.isArray(input)) {
+    input = [input];
+  }
+
+  // Instantiates a client
+  const translate = Translate();
+
+  const options = {
+    to: target,
+    // Make sure your project is whitelisted. Please refer to the upstream
+    // documentation for possible values.
+    model: model
+  };
+
+  // Translates the text into the target language. "input" can be a string for
+  // translating a single piece of text, or an array of strings for translating
+  // multiple texts.
+  return translate.translate(input, options)
+    .then((results) => {
+      let translations = results[0];
+      translations = Array.isArray(translations) ? translations : [translations];
```

Comment:
Personally, I feel that using a library for something like this is taking the Node.js "module all the things" too far...